xml/ha_hawk2_manage_i.xml

```diff
@@ -0,0 +1,365 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+ xmlns:xi="http://www.w3.org/2001/XInclude" 
+ xmlns:xlink="http://www.w3.org/1999/xlink" 
+ version="5.0" xml:id="sec.conf.hawk2.manage">
+  
+  <title>Managing Cluster Resources</title>
+  
+    <info>
+   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:maintainer/>
+    <dm:status>editing</dm:status>
+    <dm:deadline/>
+    <dm:priority/>
+    <dm:translation>yes</dm:translation>
+    <dm:languages/>
+    <dm:release/>
+    <dm:repository/>
+   </dm:docmanager>
+    </info>
+ 
+  <para> In addition to configuring your cluster resources, &hawk; allows you to manage existing
+   resources from the <guimenu>Status</guimenu> screen. For a general overview of the screen refer
+   to <xref linkend="sec.conf.hawk2.manage.monitor.status"/>.</para>
+  <sect2 xml:id="sec.conf.hawk2.manage.start">
+   <title>Starting Resources</title>
+   <para> Before you start a cluster resource, make sure it is set up correctly. For example, if you
+    want to use an Apache server as a cluster resource, set up the Apache server first. Complete the
+    Apache configuration before starting the respective resource in your cluster. </para>
+   <note>
+    <title>Do Not Touch Services Managed by the Cluster</title>
+    <para> When managing a resource via the &hasi;, the same resource must not be started or
+     stopped otherwise (outside of the cluster, for example manually or on boot or reboot). The
+     &hasi; software is responsible for all service start or stop actions. </para>
+    <para> However, if you want to check if the service is configured properly, start it manually,
+     but make sure that it is stopped again before &ha; takes over. </para>
+    <para> For interventions in resources that are currently managed by the cluster, set the
+     resource to <literal>maintenance mode</literal> first. <remark>taroth 2015-12-04: krig - how
+      does that work with hawk2? does it work already? I searched in the Operations column for a
+      resource but could not find anything there...</remark>
```

Comment:
Hm, sure? I thought 'unmanaged' and 'maintenance mode' are two different things... see https://www.suse.com/documentation/sle-ha-12/book_sleha/data/sec_ha_config_basics_resources.html#sec_ha_config_basics_meta_attr